srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/AdminHomeController.java

```diff
@@ -1,4 +1,4 @@
-package gov.va.med.srcalc.web.controller;
+package gov.va.med.srcalc.web.controller.admin;
```

Comment:
Should we also make a "gov.va.med.srcalc.web.controller.admin.controller" package?